lib/models/news.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:app/models/Explorable.dart';
```

Comment:
Sorry, I should have mentioned, we currently have a class called "Article" that handles News stuff. That should match the api requirements.

Reply:
I think Article is fine! Should I change everything else to Article (e.g. ExploreArticleTile) or leave it as is?

Reply:
Yes please!